resources/definitions/fdmprinter.def.json

```diff
@@ -5127,6 +5127,18 @@
                         }
                     }
                 },
+                "support_infill_density_multiplier_layer_0":
+                {
+                    "label": "Support Infill Density Multiplier First Layer",
+                    "description": "Multiplier for the infill on the first layers of the support. Increasing this may help for bed adhesion.",
```

Comment:
In the same vein: It's probably a bit unclear to the user what is meant by 'first'.